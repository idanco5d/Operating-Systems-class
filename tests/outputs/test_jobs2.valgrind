==8639== Memcheck, a memory error detector
==8639== Copyright (C) 2002-2022, and GNU GPL'd, by Julian Seward et al.
==8639== Using Valgrind-3.20.0 and LibVEX; rerun with -h for copyright info
==8639== Command: /mnt/c/Users/illus/os/hw/1/smash
==8639== Parent PID: 8634
==8639== 
==8639== 
==8639== FILE DESCRIPTORS: 4 open (3 std) at exit.
==8639== Open file descriptor 3: /mnt/c/Users/illus/os/hw/1/tests/outputs/test_jobs2.valgrind
==8639==    <inherited from parent>
==8639== 
==8639== 
==8639== HEAP SUMMARY:
==8639==     in use at exit: 0 bytes in 0 blocks
==8639==   total heap usage: 85 allocs, 85 frees, 82,285 bytes allocated
==8639== 
==8639== All heap blocks were freed -- no leaks are possible
==8639== 
==8639== For lists of detected and suppressed errors, rerun with: -s
==8639== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
