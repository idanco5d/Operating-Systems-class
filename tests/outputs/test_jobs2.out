smash> smash> smash> [1] sleep 10& : 9295 0 secs
smash> smash> [1] sleep 10& : 9295 0 secs
[2] sleep 20& : 9296 0 secs
smash> smash> smash> [1] sleep 10& : 9295 0 secs
[2] sleep 20& : 9296 0 secs
[3] sleep 30& : 9297 0 secs
[4] sleep 40& : 9298 0 secs
smash> smash: sending SIGKILL signal to 4 jobs:
9295: sleep 10&
9296: sleep 20&
9297: sleep 30&
9298: sleep 40&
